jbehave-support-core/docs/Expression-commands.md

```diff
@@ -185,16 +185,27 @@ This command simply do the sum of parameters.
 > Result of the command: `7`
 
 #### RANDOM_DATE
-Generates random date in range 1970 - 2059.
+Generates random date in range 1970 - 2059 by default.
+Takes up to 2 optional parameters to specify start and/or end date in several formats:
+- YYYY
+- YYYY.M
```

Comment:
Any specific reason for dot separator? I think it might be better to be consistent and use `-` since we already use it in FORMAT_DATE for example

Reply:
We use anything use provides: `DateTimeFormatter.ofPattern(dateFormat);`
But it is generally good question to establish some common approach to date formatting. Maybe pattern defined in properties, defaulting to some ISO?